.github/workflows/project_issues.yml

```
@@ -1,18 +1,83 @@
-name: Project Automation - move issues
+name: Project Automation - Move Issues
 
 on:
+  pull_request:
+    types: [opened, synchronize, reopened]
   issues:
     types: [labeled]
 
+env:
+  PAT_TOKEN: ${{ secrets.PAT_TOKEN }}
+  PROJECT_URL: https://github.com/orgs/Royal-Navy/projects/9
+
 jobs:
-  Move_labelled_issues:
+  Test_issue_mover:
     runs-on: ubuntu-latest
+    if: github.event_name == 'pull_request'
     steps:
-      - name: Move small/med labeled issues to Candidates column
-        uses: Royal-Navy/design-system-moveissue-action@master
+      - name: Checkout repository
+        uses: actions/checkout@v3
```

Would be good to update to v4 here and below:

```suggestion
        uses: actions/checkout@v4
```